sw/device/lib/arch/device_fpga_cw310.c

```diff
@@ -13,7 +13,11 @@
 
 const device_type_t kDeviceType = kDeviceFpgaCw310;
 
-const uint64_t kClockFreqCpuHz = 10 * 1000 * 1000;  // 10MHz
+const uint64_t kClockFreqCpuMhz = 10;
+
+const uint64_t kClockFreqCpuHz = kClockFreqCpuMhz * 1000 * 1000;  // 10MHz
```

Comment:
nit: 10MHz is now clear from the `kClockFreqCpuMhz` above (same goes for other `Hz` measurements)
```suggestion
const uint64_t kClockFreqCpuHz = kClockFreqCpuMhz * 1000 * 1000; 
```

Reply:
Done